contrib/src/main/java/org/apache/apex/malhar/contrib/kudu/BaseKuduOutputOperator.java

```diff
@@ -65,11 +67,27 @@
 
   public BaseKuduOutputOperator() throws IOException, ClassNotFoundException
   {
+    windowDataManager = new FSWindowDataManager();
+    initConnectionBuilderProperties(DEFAULT_CONNECTION_PROPS_FILE_NAME);
+  }
+
+  public BaseKuduOutputOperator(WindowDataManager windowManager, String configFileInClasspath)
```

Comment:
There was already a getter/setter in the abstract class for managing the window manager.  The default implementation extends from this abstract class.